<launch>
  <master auto="start"/>
  <param name="/use_sim_time" value="true"/>

  <arg name="icp_succ" default="true" />
  <arg name="image_debug" default="false" />
  <arg name="debug" default="false" />
  <arg unless="$(arg debug)" name="launch_prefix" value="" />
  <!-- <arg     if="$(arg debug)" name="launch_prefix" value="gdb &#45;&#45;ex run &#45;&#45;args" /> -->
  <arg if="$(arg debug)" name="launch_prefix" value="xterm -e gdb --args" />

  <node pkg="sbpl_perception" type="experiments" name="experiments" output="screen" launch-prefix="$(arg launch_prefix)" respawn="false">
    <param name="image_debug" value="$(arg image_debug)"/>
    <param name="icp_succ" value="$(arg icp_succ)"/>

    <!-- 4 glasses-->
    <param name="config_file" value="$(find sbpl_perception)/data/experiment_input/frame_20111220T114827.329624.txt"/>
    <!-- <param name="config_file" value="$(find sbpl_perception)/data/experiment_input/frame_20111221T142413.808534.txt"/> -->
  </node>
</launch>
